terminus-ui/toggle/src/toggle.component.ts

```diff
@@ -104,7 +104,18 @@ export class TsToggleComponent extends TsReactiveFormBaseComponent {
    * Emit an event each time the toggle value changes
    */
   // TODO: Rename to avoid conflict with native events: https://github.com/GetTerminus/terminus-ui/issues/1472
+  /**
+   * @deprecated change will be deprecated, use 'selectionChange' instead
```

Comment:
We don't need to actually say the item will be deprecated. The `@deprecated` annotation will get printed out by the linter so we just need our custom message after. eg

`@deprecation Use 'selectionChange' instead`

Will print out as

`'change' is deprecated. Use 'selectionChange' instead'`